lib/squadster_web/controllers/session_controller.ex

```diff
@@ -20,16 +20,16 @@ defmodule SquadsterWeb.SessionController do
   def callback(%{assigns: %{ueberauth_auth: auth}} = conn, _params) do
     case User.find_or_create(auth) do
       {:ok, user} ->
-        conn
-        |> put_status(:created)
-        |> json(%{message: "Logged in", token: user.auth_token})
+        redirect(conn, external: "#{auth_callback_url}?message=logged_in&token=#{user.auth_token}")
```

Comment:
Mega Javist told that `#{auth_callback_url}?message=logged_in&token=#{user.auth_token}` can be moved to const

Reply:
Spravedlivo, but Elixir doesn't have constants as in java))0
I left it as is in order to replace with some lib for params parsing later. To be something like:
`redirect(conn, external: Url(auth_callback_url, params: %{...}`)

But that makes sense, will move to module variable